packages/shortcuts-extension/package.json

```
@@ -44,13 +44,13 @@
     "@jupyterlab/settingregistry": "^4.0.0-alpha.13",
     "@jupyterlab/translation": "^4.0.0-alpha.13",
     "@jupyterlab/ui-components": "^4.0.0-alpha.28",
-    "@lumino/algorithm": "^2.0.0-alpha.1",
-    "@lumino/commands": "^2.0.0-alpha.1",
-    "@lumino/coreutils": "^2.0.0-alpha.1",
-    "@lumino/disposable": "^2.0.0-alpha.1",
-    "@lumino/domutils": "^2.0.0-alpha.1",
+    "@lumino/algorithm": "^2.0.0-alpha.4",
+    "@lumino/commands": "^2.0.0-alpha.4",
+    "@lumino/coreutils": "^2.0.0-alpha.4",
+    "@lumino/disposable": "^2.0.0-alpha.4",
+    "@lumino/domutils": "^2.0.0-alpha.4",
     "@lumino/keyboard": "^1.8.1",
```

It seems we missed that one in the first PR:

```suggestion
    "@lumino/keyboard": "^2.0.0-alpha.4",
```